egs/librispeech/ASR/pruned_transducer_stateless7/zipformer.py

```diff
@@ -1258,16 +1258,26 @@ def multi_head_attention_forward(
         # the following .as_strided() expression converts the last axis of pos_weights from relative
         # to absolute position.  I don't know whether I might have got the time-offsets backwards or
         # not, but let this code define which way round it is supposed to be.
-        pos_weights = pos_weights.as_strided(
-            (bsz, num_heads, seq_len, seq_len),
-            (
-                pos_weights.stride(0),
-                pos_weights.stride(1),
-                pos_weights.stride(2) - pos_weights.stride(3),
-                pos_weights.stride(3),
-            ),
-            storage_offset=pos_weights.stride(3) * (seq_len - 1),
-        )
+        if torch.jit.is_tracing():
```

Comment:
Please re-check this file.
There is a bunch of `torch.jit.is_scripting()` in this file. Please also add `torch.jit.is_tracing()`.

That is, change
```
if torch.jit.is_scripting()
```
to
```
if torch.jit.is_scripting() or torch.jit.is_tracing()
```

Reply:
Could you also change `scaling.py`? For instance, the following one:

https://github.com/k2-fsa/icefall/blob/b1f673aa2ce594062ace7e7563fbe329db83c6d1/egs/librispeech/ASR/pruned_transducer_stateless7/scaling.py#L793

Reply:
And this one
https://github.com/k2-fsa/icefall/blob/b1f673aa2ce594062ace7e7563fbe329db83c6d1/egs/librispeech/ASR/pruned_transducer_stateless7/scaling.py#L264